products/fairphone.md

```diff
@@ -0,0 +1,38 @@
+---
+permalink: /fairphone
+title: Fairphone
+category: device
+releasePolicyLink: https://forum.fairphone.com/t/fairphone-product-release-cycle/52652
+discontinuedColumn: true
+activeSupportColumn: false
+releaseColumn: false
+releaseDateColumn: true
+releaseLabel: "Fairphone __RELEASE_CYCLE__"
+releases:
+-   releaseCycle: "4"
+    discontinued: false
+    eol: false
+    releaseDate: 2021-09-30
+-   releaseCycle: "3+"
```

Comment:
could you please add spaces between releasecycles

Reply:
:heavy_check_mark: Done